tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs

```diff
@@ -15,14 +15,17 @@ public class VstsBuildBatchUpdate
         readonly DevOpsAccessSetting devOpsAccessSetting;
         readonly string dbConnectionString;
         Dictionary<string, Dictionary<BuildDefinitionId, DateTime>> lastUpdatePerBranchPerDefinition;
+        HashSet<string> branches;
 
-        public VstsBuildBatchUpdate(DevOpsAccessSetting devOpsAccessSetting, string dbConnectionString)
+        public VstsBuildBatchUpdate(DevOpsAccessSetting devOpsAccessSetting, string dbConnectionString, HashSet<string> branches)
         {
             ValidationUtil.ThrowIfNull(devOpsAccessSetting, nameof(devOpsAccessSetting));
+            ValidationUtil.ThrowIfNulOrEmptySet(branches, nameof(branches));
```

Comment:
Typo in ```Nul```?

Also, I think we should do a test run to make sure this works.

Reply:
I ran the solution; just the method name has typo.